TestPlaybooks/playbook-EmailReputationTest.yml

```diff
@@ -277,7 +277,7 @@ tasks:
       brand: ""
     scriptarguments:
       message:
-        simple: Didn't get proper score for indicator
+        simple: 'Didn''t get proper score for indicator. Expected: 2, Got: ${res_score.DBotScore.[0].Score}'
```

Comment:
```suggestion
        simple: Didn't get proper score for indicator. Expected: 2, Got: ${res_score.DBotScore.[0].Score}'
```

Reply:
Nice

Reply:
this is how the playbook were downloaded from demisto

Reply:
🤷‍♂